AbstractScreen.java

```diff
@@ -184,12 +184,27 @@ protected JButton newButton(String text, int x, int y, Runnable onClick) {
     return newButton(text, Fonts.MEDIUM, new Bounds(x, y, STD_BUTTON_WIDTH, STD_BUTTON_HEIGHT),
         onClick);
   }
-
-  protected LabeledComponent<JTextField> newTextField(String label, Font font, Bounds bounds) {
+  protected TextFieldWithPrompt newTextFieldWithPrompt(String label, Font font, Bounds bounds) {
     final var jtf = new TextFieldWithPrompt(WIDTH / 15, label);
     jtf.setBounds(bounds);
     jtf.setFont(font);
-    return new LabeledComponent<>(jtf, "", this);
+    add(jtf);
+    return jtf;
+  }
+
+  protected TextFieldWithPrompt newTextFieldWithPrompt(String label, Font font, int x, int y) {
+    return newTextFieldWithPrompt(label, font, new Bounds(x, y, STD_TEXT_FIELD_WIDTH, STD_TEXT_FIELD_HEIGHT));
+  }
+
+  protected TextFieldWithPrompt newTextFieldWithPrompt(String label, Font font, int y) {
+    return newTextFieldWithPrompt(label, font, centeredX(STD_TEXT_FIELD_WIDTH), y);
+  }
+
+  protected LabeledComponent<JTextField> newTextField(String label, Font font, Bounds bounds) {
+    final var jtf = new JTextField(WIDTH / 15);
+    jtf.setBounds(bounds);
+    jtf.setFont(Fonts.SMALL);

```

Comment:
this should use the given font

Reply:
actually `jtf.setFont(Fonts.SMALL);` part was originally there.
lemme fix that too

Reply:
oops, can you make a new pr

Reply:
got'ya